spec/mailers/notifier_spec.rb

```diff
@@ -478,7 +478,11 @@
       expect(@confirm_email.to).to eq([bob.unconfirmed_email])
     end
 
-    it "has the unconfirmed emil in the subject" do
+    it "FROM: header should be the default sender address" do
+      expect(@confirm_email["From"].to_s).to eq(AppConfig.mail.sender_address.to_s)
```

Comment:
We did this because the checks before "complained" about this. but we could change it if you prefer